scripts/select_appointments.sql

```diff
@@ -1 +1,15 @@
 -- retrieve appointment information
+-- Retrieve the names and appointment times for everyone seeing Dr. Hubbard.
+SELECT p.given_name, p.surname
+FROM patients p
+INNER JOIN appointments a
+  ON a.date = a.date
```

Comment:
Almost there!
In order for this to work you need to select `p` from `patients` joined with appointments `a` (for appointments where a.patient_id is equal to p.id) joined with doctors `d` (for appointments in which a.doctor_id  d.id) where the doctor's surname is 'Hubbard'.